spec/bolt/cli_spec.rb

```diff
@@ -165,6 +165,68 @@ def stub_config(file_content = {})
     end
   end
 
+  context 'module' do
+    include BoltSpec::Project
+
+    let(:cli)     { Bolt::CLI.new(command + %W[--project #{project_path}]) }
+    let(:command) { ['module'] }
+
+    around(:each) do |example|
+      original = ENV['BOLT_MODULE_FEATURE']
+      ENV['BOLT_MODULE_FEATURE'] = 'true'
+
+      with_project do
+        example.run
+      end
+    ensure
+      ENV['BOLT_MODULE_FEATURE'] = original
+    end
+
+    before(:each) do
+      allow(Bolt::Project).to receive(:new).and_return(project)
+    end
+
+    it 'errors without BOLT_MODULE_FEATURE being set' do
+      ENV.delete('BOLT_MODULE_FEATURE')
+      expect { cli.parse }.to raise_error(Bolt::CLIError)
+    end
+
+    context 'without modules configured' do
```

Comment:
I don't think we need to wrap this in a context, the `it` should just have a more verbose message.